packages/hooks/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./useResizeObserver";
```

Comment:
Currently we have this at the root and exporting everything. I see that this is how we export in `@jobber/formatter` but not in `@jobber/components`. My assumption is that this is fine and that we will just export everything from the root. Just want to make sure that this is intentional and that we dont want to do something like:

```js
import { useResizeObserver } from "@jobber/hooks/useResizeObserver"
```

Reply:
Yeah, that's a fair question. @seanhealy Any advice on which of the two organizing structures we want here?

Reply:
We could also ship this way, and iterate. Right now, making the switch wouldn't be hard.

Reply:
Yea, I'm not sure on this one either. I can see the upsides of both ways…

Reply:
After talking with Jeff I think we're better off leaving this as is for now. the `hooks/foo` syntax requires a bit of fiddly build time configuration that I don't think we'll really benefit from doing right now.